tests/reject-tests.sh

```diff
@@ -26,45 +26,11 @@ sed 's/localhost/'$SERVER_IP'/g' <$srcdir/servers >servers-temp$PID
 
 echo ../src/radiusclient -D -i -f radiusclient-temp$PID.conf  User-Name=admin Password=admin | tee $TMPFILE
 ../src/radiusclient -D -i -f radiusclient-temp$PID.conf  User-Name=admin Password=admin | tee $TMPFILE
-if test $? != 0;then
-	echo "Error in PAP auth"
-	exit 1
-fi
-
-grep "^Framed-Protocol                  = 'PPP'$" $TMPFILE >/dev/null 2>&1
-if test $? == 0;then
-	echo "Error in data received by server (Framed-Protocol)"
-	cat $TMPFILE
-	exit 1
-fi
-
-grep "^Framed-IP-Address                = '192.168.1.190'$" $TMPFILE >/dev/null 2>&1
-if test $? == 0;then
-	echo "Error in data received by server (Framed-IP-Address)"
-	cat $TMPFILE
-	exit 1
-fi
-
-grep "^Framed-Route                     = '192.168.100.5/24'$" $TMPFILE >/dev/null 2>&1
 if test $? == 0;then
-	echo "Error in data received by server (Framed-Route)"
-	cat $TMPFILE
+	echo "Authentication passed. Unexpected"
```

Comment:
It seems here you address my previous comment. Thank you. I'd suggest to combine the changes to 2-3 targetted commits.